recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index b538d15..7c811a8 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -47,6 +47,10 @@ target_link_libraries(${OATPP_THIS_MODULE_NAME}
+         PUBLIC ${PostgreSQL_LIBRARIES}
+ )
+ 
++if(MSVC)
```

Comment:
Can this patch be upstreamed?

Reply:
I will create PR for this.

Reply:
merged.
https://github.com/oatpp/oatpp-postgresql/pull/15

Reply:
If it's good enough for upstream, it's more then okay for us 🤣